bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/GF_HHH_benchmarks/GF_HHH_c3_0_d4_0/GF_HHH_c3_0_d4_0_run_card.dat

```diff
@@ -0,0 +1,268 @@
+ 0.0   = htjmin ! minimum jet HT=Sum(jet pt)
+ -1.0  = htjmax ! maximum jet HT=Sum(jet pt)
+ 0.0   = ihtmin  !inclusive Ht for all partons (including b)
+ -1.0  = ihtmax  !inclusive Ht for all partons (including b)
+ 0.0   = ht2min ! minimum Ht for the two leading jets
+ 0.0   = ht3min ! minimum Ht for the three leading jets
+ 0.0   = ht4min ! minimum Ht for the four leading jets
+ -1.0  = ht2max ! maximum Ht for the two leading jets
+ -1.0  = ht3max ! maximum Ht for the three leading jets
+ -1.0  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+ 0.0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ True = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0.0   = xetamin ! minimum rapidity for two jets in the WBF case
+ 0.0   = deltaeta ! minimum rapidity for two jets in the WBF case
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1.0    =  ktdurham
+ 0.4  =  dparameter
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0.0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+   True  = use_syst      ! Enable systematics studies
+#
```

Comment:
Hi @mlizzo ,

Thanks a lot for having a look. So I tried to copy the run_card you suggested but I don't manage to make it run, in Madgraph stand-alone or in the gridpack generation using the CMS software (located in genproduction). 

In principle, I believe I would need additional work in the model to make it work for NLO. I'm not an expert here so correct me if I'm wrong. However, what I did for validation was to cross-check this implementation with a theory paper. 

I'm attaching some slides here, let me know what you think! 

Cheers,

Marko

[20220607_HHH_hig-mc.pdf](https://github.com/cms-sw/genproductions/files/8920880/20220607_HHH_hig-mc.pdf)

Reply:
Hi @mstamenk,

Do you want a LO or NLO sample? Because you added the [QCD] syntax in your proc_card, that's why I suggested you to update the run_card. By looking at the presentation you attached it seems to me you want a LO production instead, in that case the run_card you have should be fine. Could you please clarify?

Reply:
Hi @mlizzo ,

I would like a LO production. I dont think there's a working implementation for a NLO implementation, unless I'm missing something. Let me know if you think it's possible to get the NLO. 

Cheers,

Marko

Reply:
Hello,

Just a friendly reminder about this PR. Let me know if you need anything else from my side.